cost-analyzer/values.yaml

```diff
@@ -45,7 +45,7 @@ global:
 
   # Amazon Managed Service for Prometheus
   amp:
-    enabled: false  # If true, kubecost will be configured to remote_write and query from Amazon Managed Service for Prometheus.
```

Comment:
we want to keep this off by default, yeah? otherwise LGTM

Reply:
Forgot to revert.

Reply:
Fixed.